ppr-ui/src/resources/userAccessOrgLookup.ts

```diff
@@ -3,16 +3,15 @@ import { MhrSubTypes } from '@/enums'
 
 export const UserAccessOrgLookupConfig: Record<MhrSubTypes, OrgLookupConfigIF> = {
   [MhrSubTypes.LAWYERS_NOTARIES]: {
```

Comment:
Should the enum be updates as well? Because the title changed.

Reply:
I'm not sure what you mean, is this in reference to the pluralTitle property value changing?
If so, i did ask Tracey explicitly about this one, for Lawyers and Notaries, design wanted this particular label to be generic (ie Qualified Suppliers).

Reply:
Yea it was a reference to the `pluralTitle`. Sounds like it's a harmless content change.